apps/user/views/admin/enrollments.py

```diff
@@ -0,0 +1,217 @@
+from rest_framework import status
+from rest_framework.permissions import IsAdminUser
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from apps.user.models import (
+    CohortStudent,
+    EnrollmentStatus,
+    StudentEnrollmentRequest,
+    User,
+)
+from apps.user.models.user import RoleChoices
+from apps.user.models.withdraw import Withdrawal
+from apps.user.pagination import AdminAccountPagination
+from apps.user.serializers.admin.enrollments import (
+    AdminStudentEnrollAcceptSerializer,
+    AdminStudentEnrollRejectSerializer,
+    AdminStudentEnrollRequestSerializer,
+    AdminStudentEnrollSerializer,
+    AdminStudentSerializer,
+)
+
+ORDERING_MAP = {"id": "id", "latest": "-created_at", "oldest": "created_at"}
+
+USER_STATUS_FILTERS = {
+    "withdrew": Q(is_withdrawing=True),
+    "activated": Q(is_withdrawing=False, is_active=True),
+    "deactivated": Q(is_withdrawing=False, is_active=False),
+}
+
+
+ENROLLMENT_STATUS_FILTERS = {
+    "pending": Q(status=EnrollmentStatus.PENDING),
+    "accepted": Q(status=EnrollmentStatus.ACCEPTED),
+    "rejected": Q(status=EnrollmentStatus.REJECTED),
+    "canceled": Q(status=EnrollmentStatus.CANCELED),
+}
+
+
+class AdminStudentView(APIView):
+    permission_classes = [IsAdminUser]
+
+    @extend_schema(
+        tags=["회원관리"],
+        summary="수강생 목록 조회 API",
+        parameters=[
+            OpenApiParameter("page", OpenApiTypes.INT, required=False, location=OpenApiParameter.QUERY),
+            OpenApiParameter("page_size", OpenApiTypes.INT, required=False, location=OpenApiParameter.QUERY),
+            OpenApiParameter("search", OpenApiTypes.STR, required=False, location=OpenApiParameter.QUERY),
+            OpenApiParameter(
+                "status",
+                OpenApiTypes.STR,
+                required=False,
+                location=OpenApiParameter.QUERY,
+                enum=["activated", "deactivated", "withdrew"],
+            ),
+            OpenApiParameter("course_id", OpenApiTypes.INT, required=False, location=OpenApiParameter.QUERY),
+            OpenApiParameter("cohort_number", OpenApiTypes.INT, required=False, location=OpenApiParameter.QUERY),
+        ],
+    )
+    def get(self, request: Request) -> Response:
+        students = User.objects.filter(role=RoleChoices.ST).order_by("id")
+        students = students.annotate(is_withdrawing=Exists(Withdrawal.objects.filter(user_id=OuterRef("pk"))))
+
+        course_id = request.query_params.get("course_id")
+        cohort_number = request.query_params.get("cohort_number")
+
+        if course_id and str(course_id).isdigit():
+            students = students.filter(cohortstudent__cohort__course_id=int(course_id))
+
+            if cohort_number and str(cohort_number).isdigit():
+                students = students.filter(cohortstudent__cohort__number=int(cohort_number))
```

Comment:
```suggestion
        course_id = request.query_params.get("course_id", "")
        cohort_number = request.query_params.get("cohort_number", "")

        if course_id.isdigit():
            students = students.filter(cohortstudent__cohort__course_id=int(course_id))

            if cohort_number.isdigit():
                students = students.filter(cohortstudent__cohort__number=int(cohort_number))
```
isdigit이 문자열이 숫자로 이루어져있는지 검증하는 거기 때문에 
문자열에 값이 있음을 포함하고 있어서 이렇게 간소화할 수 있지 않을까요?